src/image-default.yaml

```diff
@@ -1,2 +1,4 @@
 # This file contains defaults for image.yaml that is used by create_disk.sh
+bootfs: "ext4"
 rootfs: "xfs"
+grub-script: "/usr/lib/coreos-assembler/grub.cfg"
```

Comment:
Does this even need to be configurable? Not sure why it was a knob in the first place.

Reply:
I think the original idea was that `create_disk.sh` is theoretically a distinct thing from cosa; i.e. you could run it outside.